### PR TITLE
extension commit

### PR DESCRIPTION
**Pull request by caseyoppenheim**
Browser extension "Casey" with simple Hello World statement

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (wlycdgr) [inline .gitignore]: This lil red stop sign is GitHub's way of complaining that there's no newline at the end of the file. Always putting a newline at the end of every file is a good habit to get into because it can minimize the risk of file parsing / processing snafus on *nix machines. Here's one good discussion: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Casey's Extension",
+    "version": "1",
+    "description": "Hello World!",
+    "permissions": ["activeTab"],
+    "manifest_version": 2,
```
Comment (wlycdgr) [inline manifest.json]: By convention, `manifest_version` should be the first entry in the manifest

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Casey's Extension",
+    "version": "1",
```
Comment (wlycdgr) [inline manifest.json]: Using semantic versioning (major.minor.bugfix/tweak, e.g. "8.4.2") is a good habit to get into. Major version bumps break compatibility with previous major versions. Minor version bumps add new features to the current major version without breaking backward compatibility. Bugfix/tweak version bumps fix issues with the existing features and modify/improve them.

Tl;dr: Go with "1.0.0" :)

Ts;wrm: https://semver.org/

File: manifest.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    "description": "Hello World!",
+    "permissions": ["activeTab"],
+    "manifest_version": 2,
+    "browser_action": {
+        "default_popup": "popup.html",
+        "default_icon": "casey_project.png",
```
Comment (wlycdgr) [inline manifest.json]: Nice icon ~
It's good practice to give your image (and other) files names that describe what they are and/or what purpose they serve. So here, a better name would be something like `browserIcon32.png`.
Also, it's good practice to keep your project resources organized into folders by type. Really helps things from getting out of hand disorganized once a project starts getting any kind of sizeable. Go ahead and make an `images` folder, chuck the icon file in there, and update the path here in the manifest accordingly. (Look at our extension manifest for an example if you need to).

**Review by wlycdgr — COMMENTED** [review]
Take a look at the comments & lemme know if you have any questions ~ Good work so far!

File: manifest.json [hunk 5]
```diff
@@ -1,9 +1,10 @@
 {
+    "manifest_version": 2,
     "name": "Casey's Extension",
-    "version": "1",
+    "version": "1.1.0",
```
Comment (wlycdgr) [inline manifest.json]: Here, we wanna just go with "1.0.0". We only increment the version when there's a public release - not as part of the process of creating and editing code in preparation for one. For the purposes of this project, we will have just the one 'release' at the end, when the vertical slice is done.

File: .gitignore (hunk 1 above)
Comment (wlycdgr) [reply .gitignore]: Remember to add this new line over here, either manually or by installing the VSCode extension Frank mentioned in our groupchat.

File: popup.html [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  }
+  h1{
+      font-size: 12px;
+      text-align: center;
+  }
+    </style>
+<body>
+   <h1>Hello, World!</h1> 
+  </body>
+</html>
```
Comment (wlycdgr) [inline popup.html]: Throw a newline in at the end of this file too please ~

**Review by wlycdgr — APPROVED** [review]
